src/components/shared/VirtualList.js

```diff
@@ -129,7 +129,7 @@ class VirtualListInner extends React.PureComponent<VirtualListInnerProps> {
         className={className}
         ref={this._takeContainerRef}
         style={{
-          height: `${items.length * itemHeight}px`,
+          height: `${items.length * itemHeight + 20}px`,
```

Comment:
What do you think about making this leave an extra row at the bottom like this:

```
           height: `${(items.length + 1) * itemHeight}px`, 
```

This has the advantage of not being a magic number, like 20, but I'm assuming this is probably a little bit smaller than the privacy notice overlay. I'm personally OK with ignoring the privacy notice size, as we may be moving that notice anyway.

If you feel that we should hard code a value here, then please create a variable binding at the top of the file like:

```
const VIRTUAL_LIST_BOTTOM_PADDING = 20;
```